.github/workflows/ddp.yaml

```diff
@@ -13,9 +13,9 @@ jobs:
       matrix:
         python-version: ['3.9']
         pytorch-version: [
-          'torch==1.13.1+cu117 torchvision==0.14.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117',
-          'torch==2.0.0+cu117 torchvision==0.15.1+cu117 --index-url https://download.pytorch.org/whl/cu117',
-          'torch==2.0.1+cu117 torchvision==0.15.2+cu117 --index-url https://download.pytorch.org/whl/cu117'
+          'torch==1.13.1+cu117 torchvision==0.14.1+cu117 torchaudio==0.13.1 --extra-index-url https://download.pytorch.org/whl/cu117',
+          'torch==2.0.0+cu117 torchvision==0.15.1+cu117 torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cu117',
+          'torch==2.0.1+cu117 torchvision==0.15.2+cu117 torchaudio==2.0.2+cu117 --index-url https://download.pytorch.org/whl/cu117'
```

Comment:
Do we need torchaudio?

Reply:
it seems that you are importing torchaudio in torchmetric.
 Previously, there was a failure in the action due to version mismatch between torchaudio and torch.

Reply:
Additionally, the same issue was observed on Windows, and it was resolved by changing the versions of torchaudio, numpy, and numba.